### PR TITLE
links2: add page

### PR DESCRIPTION
**Pull request by knqti**
- [ ] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [ ] The page(s) have at most 8 examples.
- [ ] The page description(s) have links to documentation or a homepage.
- [ ] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [ ] The page(s) follow the [style guide](/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md).
- [ ] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message-and-pr-title).
- **Version of the command being documented (if known):**

### CODE REVIEW

File: pages/linux/links2.md [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+
+- Visit a website in graphics mode:
+
+`links2 -g {{https://example.com}}`
+
+- Apply restrictions for anonymous account:
+
+`links2 -anonymous {{https://example.com}}`
+
+- Enable Cookies (`1` to enable):
```
Comment (spageektti) [inline pages/linux/links2.md:18]:
```suggestion
- Toggle Cookies (`1` to enable):
```

File: pages/linux/links2.md [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+> Command-line web browser.
+> More information: <http://links.twibright.com/>.
+
+- Visit a website:
+
+`links2 {{https://example.com}}`
+
+- Visit a website in graphics mode:
+
+`links2 -g {{https://example.com}}`
```
[the remark this answers is not in the document]
Comment (knqti) [reply pages/linux/links2.md:12]: Running `links -g` gives the error: 

> Graphics not enabled when compiling (use links2 instead for graphics mode)

File: pages/linux/links2.md [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# links2
+
+> Command-line web browser.
```
Comment (spageektti) [inline pages/linux/links2.md:3]:
```suggestion
> Command-line web browser.
> See also: `links`.
```

**Comment by knqti** [comment]
> Since links2 seems to me like v2 of links, I miss a reference to `links`, or specify how this page differs from `links.md`.

I believe graphics mode is the biggest difference between `links2` and `links`. Can I leave the graphics mode explanation and then refer to `links` per @spageektti's suggestion?

> See also: `links`.

**Review by spageektti — APPROVED** [review]

**Pull request merged**